__test__/src/app.test.js

```diff
@@ -0,0 +1,24 @@
+const { app, startServer } = require('../../dist/src/app.js');
+
+
+
+describe('Server POL', () => {
+  it('starts with port', () => {
+    jest.spyOn(global.console, 'log');
+    startServer(3000)
+      .then((server) => {
+        server.close(() => {
+          expect(console.log).toHaveBeenCalledWith('Listening on 3000')
+        })
+      })
+  });
+  it('starts without port', () => {
+    jest.spyOn(global.console, 'log');
+    startServer()
+      .then((server) => {
+        server.close(() => {
+          expect(console.log).toHaveBeenCalledWith('Listening on 3001')
+        });
+      })
+  })
```

Comment:
Would like to see more tests in here, tests that would be great to have:
- Can a user sign up
- Can a user sign in
- User can hit routes that they have access to
- Warning if user tries to get to route that they don't have access to
- Any middleware that you may have